terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/FunctionTestSupport.java

```diff
@@ -174,7 +176,10 @@ protected void bootDefaultWebDriver() {
     }
 
     private WebDriver newWebDriver() {
-        WebDriver webDriver = getApplicationContext().getBean(WebDriver.class);
+        FirefoxProfile profile = new FirefoxProfile();
+        profile.setPreference("browser.startup.homepage_override.mstone", "ignore");
+        profile.setPreference("network.proxy.type", 0);
+        WebDriver webDriver = new FirefoxDriver(profile);

```

Comment:
- [ ] Please add setPreference to terasoluna-gfw-functionaltest-selenium/src/test /resources/ META-INF/spring/seleniumContext.xml Because profile for browser should be change from Spring profile.